src/components/EditorPage/EditorPage.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from 'react'
+import Preview from '../Preview'
+
+export class EditorPage extends React.Component {
```

Comment:
Following https://github.com/decentraland/standards we should `export default` here too!